app/coffee/HttpController.coffee

```diff
@@ -11,22 +11,22 @@ module.exports = HttpController =
 			{project_id, user_id, first_name, last_name, email, connected_time} = ioClient.ol_context
 			client = {client_id, project_id, user_id, first_name, last_name, email, connected_time}
 			client.rooms = []
-			for name, joined of ioClient.manager.roomClients[client_id]
-				if joined and name != ""
-					client.rooms.push name.replace(/^\//, "") # Remove leading /
+			for name in Object.values(ioClient.rooms)
+				continue if name == ioClient.id
+				client.rooms.push name.replace(/^\//, "") # Remove leading /
```

Comment:
I know it won't be here long but in coffeescript this can be written as 
```coffee
for name in Object.values(ioClient.rooms) when name isnt ioClient.id
```

Reply:
Done in 09137f1fd68f7d26bfe78f90da941dbb7382927d